doc/README/Data/Container/FreeMonad.agda

```diff
@@ -51,13 +51,13 @@ prog =
   where
   open RawMonad monad using (_>>_)
 
-runState : {S X : Set} → State S ⋆ X → (S → X ⟨×⟩ S)
+runState : {S X : Set} → State S ⋆ X → (S → X Product.× S)
```

Comment:
Why do we actually need to qualify this? I'm not aware of any other clashing names in the library off the top my head?

Reply:
`Data.Container.Combinator._×_` clashes on import of that module. 
But it is enough to hide *that* binding instead... (although that's not possible in the *definition* of the container combinators, but that's neither here nor there in this `README` instance...)

Reply:
Duly hidden.